### DISCUSSION TITLE
Can Wikitraccs be used for self-hosted Sharepoint installations?

### DISCUSSION THREAD

**Discussion by DirkHauptmann** [WikiTransformationProject/wikitraccs-releases#81] [Q&A] [answered]
I hope I did not overlook it in the documentation. There you refer only to Azure Cloud end points. 

We are not using sharepoint in the Azurd-cloud and installed it locally at our company. 

Can Wikitraccs also be used for this?

**Comment by heinrich-ulbricht** [comment, marked as answer]
@DirkHauptmann SharePoint Online is the only supported target, as WikiTraccs creates "modern pages" only. In SharePoint on-premises those are either not available or not as advanced as in SharePoint Online, sorry.
